demo/grid-columns-demos.html

```diff
@@ -148,6 +145,22 @@ <h3>Hiding Columns</h3>
               root.firstElementChild.alt = rowData.item.name.first + ' ' + rowData.item.name.last;
               root.firstElementChild.src = rowData.item.picture.thumbnail;
             };
+            
+            const contextMenu = document.querySelector('vaadin-context-menu');
+            contextMenu.listenOn = document.querySelector('vaadin-button');
+            contextMenu.openOn = 'click';
+            contextMenu.renderer = function(root) {
+              root.innerHTML  = '<div style="display: flex; flex-direction: column;"></div>';
```

Comment:
we should avoid using `innerHTML` in demos, as it is generally considered unsafe.
Possible alternatives here would be using `display: block` on checkboxes:
```js
checkbox.style.display = 'block';
```

Reply:
In this case it's not that bad as the HTML doesn't include any variables. But, you're right, using `display: block` for the checkboxes might be cleaner. Then the `div` wrapper could be dropped also 👍

Reply:
Done